### PR TITLE
Fix inconsistent path layer id in GeoJSONLayer

### PR DESCRIPTION
**Pull request by Pessimistress**
For #5819

#### Change List
- Fix inconsistent sub layer id
- Documentation

### CODE REVIEW

File: docs/api-reference/layers/geojson-layer.md [hunk 1]
```diff
@@ -280,7 +280,7 @@ The GeoJsonLayer renders the following sublayers:
 
 * `polygons-fill` - a [SolidPolygonLayer](/docs/api-reference/layers/solid-polygon-layer.md) rendering all the `Polygon` and `MultiPolygon` features.
 * `polygons-stroke` - a [PathLayer](/docs/api-reference/layers/path-layer.md) rendering the outline of all the `Polygon` and `MultiPolygon` features. Only rendered if `stroked: true` and `extruded: false`.
-* `linestrings` - a [PathLayer](/docs/api-reference/layers/path-layer.md) rendering all the `LineString` and `MultiLineString` features.
+* `line-strings` - a [PathLayer](/docs/api-reference/layers/path-layer.md) rendering all the `LineString` and `MultiLineString` features.
```
Comment (felixpalmer) [inline docs/api-reference/layers/geojson-layer.md:283]: This feels wrong to me. In the GeoJson spec LineString is always a single word. With the other layers (e.g. polygon-fill) the dash separates the geometry type and method of drawing. Is there a reason why we can't instead make the sublayer have the id `linestrings`?

**Review by felixpalmer — CHANGES_REQUESTED** [review]

File: docs/api-reference/layers/geojson-layer.md (hunk 1 above)
Comment (Pessimistress) [reply docs/api-reference/layers/geojson-layer.md:283]: That will be a breaking change.

Comment (felixpalmer) [reply docs/api-reference/layers/geojson-layer.md:283]: Perhaps something to change for a future version?

Comment (Pessimistress) [reply docs/api-reference/layers/geojson-layer.md:283]: Sure. I was a little surprised that it was two words also, but did not think it was important enough to justify a breaking change.

**Review by kylebarron — APPROVED** [review]

**Pull request merged**